omni.yaml

```diff
@@ -1,5 +1,239 @@
 navigation:
   tabs:
+    - tab: "Talos"
```

Comment:
Since you added the other talos yaml file you need to remove these from the omni.yaml.

Reply:
Done, I did remove it earlier but I think when I rebase, it erased my changes